infra/cifuzz/fuzz_target.py

```diff
@@ -205,27 +222,49 @@ def check_reproducibility_and_regression(self, test_case):
     Returns:
       True if the crash was introduced by the current pull request.
     """
-    reproducible_in_pr = self.is_reproducible(test_case,
-                                              os.path.dirname(self.target_path))
+    reproducible_on_pr_build, ran_target = self.is_reproducible(
+        test_case, self.target_path)
+
+    if not ran_target:
+      raise Exception('Could not run target when checking for reproducibility.'
+                      'Please file an issue:'
+                      'https://github.com/google/oss-fuzz/issues/new.')
+
     if not self.project_name:
-      return reproducible_in_pr
+      return reproducible_on_pr_build
 
-    if not reproducible_in_pr:
+    if not reproducible_on_pr_build:
       logging.info(
           'Failed to reproduce the crash using the obtained test case.')
       return False
 
+    could_not_test_on_oss_fuzz_message = (
```

Comment:
Nit: put this into a global since it's quite long and clutters the function.

Reply:
Done.